portal/views.py

```diff
@@ -0,0 +1,27 @@
+from django.shortcuts import render
+
+from portal.forms import SignUpForm
+
+
+def index(request, position="child", popup=None):
+    if request.method == 'POST':
+        form = SignUpForm(request.POST)
+        if form.is_valid():
+            # ToDo: Spam protection - limit the number of requests before the user gets ignored
```

Comment:
If you are adding new TODOs, this should be reflected in an issue, and/or in the commit message.